fractal_tasks_core/tasks/copy_ome_zarr.py

```diff
@@ -188,18 +186,18 @@ def copy_ome_zarr(
                             f"{zarrurl_old=}, convert it to 2D, and "
                             "write it back to the new zarr file."
                         )
-                        ROI_table = ad.read_zarr(
+                        new_ROI_table = ad.read_zarr(
```

Comment:
For the record: this was a (minor) bug. When calling this task with `project_to_2D=False` (which almost never happens), we would not have a `new_ROI_table` defined, and then we would have failed at the `write_elem` command a few lines below.